XiEditor/EditView.swift

```diff
@@ -166,9 +114,15 @@ class EditView: NSView, NSTextInputClient {
         path2.fill()
         */
 
+        let defaultStyle = dataSource.styleMap.defaultStyle
+
+        // set view background
+        self.wantsLayer = defaultStyle.bgColor.alphaComponent != 0.0
+        self.layer?.backgroundColor = defaultStyle.bgColor.cgColor
```

Comment:
`editView.needsDisplay` sounds good. I'll have a look how to set the background with `NSRectFill` instead of using the layer.